.gitignore

```diff
@@ -226,4 +224,13 @@ tags
 !.vscode/extensions.json
 .history
 
+<<<<<<< HEAD
+###make empty downloads folder
+downloads/*
+!downloads/.gitkeep 
+=======
+downloads/*
+!downloads/.gitkeep
```

Comment:
Can you explain the reason .`gitkeep` is used here 
Can I use something like `.downloads` or something else?
Put forward your explanation.

Reply:
Is it correct if I add these lines ? 
`#downloads
.downloads`

Reply:
That results In a comment due to #
> Can you explain the reason .gitkeep is used here
Can I use something like .downloads or something else?

Hopefully you will answer this

Reply:
no, I mean,
" #downlaods
    .downloads

Reply:
> That results In a comment due to #
> 
> > Can you explain the reason .gitkeep is used here
> > Can I use something like .downloads or something else?
> 
> Hopefully you will answer this

I thought that code would not delete the downloads folder